docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml

```diff
@@ -840,11 +842,13 @@ ecl unpublish roxie "FindpersonService*"
         <para><emphasis role="bold">Examples</emphasis>:</para>
 
         <programlisting>ecl run thor findperson.ecl --name=findperson --input=data.xml --wait=1000
+ecl run thor findperson.ecl -I C:\MyECL\
```

Comment:
Have you tested this?  It uses common command processing with eclcc and at least originally didn't support a space after the -I.  It follows gcc style for includes.